### ISSUE TITLE
[nullable check]'from clause' should have nullable check in query expression

### ISSUE THREAD

**Issue by bysdxt** [dotnet/roslyn#67370]
**Version Used**: 
.NET SDK 7.0.202
Visual Studio 17.5.2

**Code**:
```C#
#nullable enable
using System;
using System.Linq;
using System.Collections.Generic;
namespace ConsoleApp1 {
    internal class Program {
        static IEnumerable<int> f(IEnumerable<int>? x) {
            return from v in x select v + 1;
        }
        static IEnumerable<int> h(IEnumerable<int>? x) {
            foreach (var v in x)
                yield return v + 1;
        }
        static IEnumerable<int> g(IEnumerable<int>? x) {
            return x.Select(v => v + 1);
        }
        static void Main() {
            Console.WriteLine("Hello, World!");
        }
    }
}
```

[sharplab](https://sharplab.io/#v2:EYLgtghglgdgNAExAagD4GIYFcA2OLA4CmABETAcQLABQAAgEwCMtdADCXUwHQAysARwDcrDlwAsImhTBEAzgAcIAY1IBhAPYw5G4gEEFCpiQDetEhZKwALkQBOFHJwacmAdlPnL3rgDZXAMwAPDYAfCQAZgAUXMFhAPwkAB4AlJ403pmWdB4RdhpgJABuVjDJJHJExMrWxSTIJExSWRYAvl5ZfoEhMNbhABYxTHG9oYmp6S2ZERp2RCr9JFFFEHZ1sMkpHVOZXMY5dQ1N297tGS1dsT19JADmQyN942lm5zucHkncAMpVRDXLEgAXnCJSOKWaLTOU0u4hIAFloDAoi8ThcmABOKIAIgAElUcBo4CQAOqzHAIACE2IhaLa2zOrSAA===) 

**Expected Behavior**:
All three functions f/h/g are warned about nullable problem
**Actual Behavior**:
Only function h/g are warned. No warning in function f.

**Comment by jcouv** [comment]
This is a known issue. We don't track nullability within LINQ queries. Closing as duplicate of https://github.com/dotnet/roslyn/issues/29863
